crates/kitsune_p2p/kitsune_p2p/Cargo.toml

```diff
@@ -47,6 +47,7 @@ maplit = { version = "1", optional = true }
 mockall = { version = "0.10.2", optional = true }
 
 [dev-dependencies]
+kitsune_p2p_bootstrap = { version = "0.0.3", path = "../bootstrap" }
```

Comment:
do we need the version here? there's a note in the [cargo book on development dependencies](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#development-dependencies):

> Note: When a package is published, only dev-dependencies that specify a version will be included in the published crate. For most use cases, dev-dependencies are not needed when published, though some users (like OS packagers) may want to run tests within a crate, so providing a version if possible can still be beneficial.

Reply:
```suggestion
kitsune_p2p_bootstrap = { path = "../bootstrap" }
```

Reply:
Good point, I don't think we need the version.